Cargo.toml

```diff
@@ -23,14 +23,17 @@ rayon = "1"
 reqwest = { version = "0.12", default-features = false, features = ["stream", "rustls-tls"] }
 reqwest-middleware = "0.4"
 reqwest-retry = "0.7"
+reqwest-tracing = "0.5"
```

Comment:
My guess is you may not need/want this. I found `ureq`'s tracing to be very noisy -- tracing loads of low level network details that we're unlikely to use and incur usage from.

Reply:
Yeah, I wasn't sure if I should add this but decided to start with a maximalist approach 😅. 

I'm not sure how the `ureq` tracing is setup but the `reqwest-tracing` crate just wraps each request in an individual span and decorates it with the following fields:
- http.request.method
- http.url
- url.scheme
- server.address
- server.port
- otel.kind
- otel.name
- otel.status_code
- user_agent.original
- http.response.status_code
- error.message
- error.cause_chain

Since this buildpack makes multiple concurrent requests to populate it's package list, I thought this might be useful information to surface for diagnostic purposes. But I can drop this if you don't think it would be valuable.

Reply:
Confirmed this adds useful diagnostics and not the whole low-level networking details. Going to keep this.